docs/extensions/fares-v2.md

```diff
@@ -21,21 +21,16 @@ Producers may implement Fares v2 in the same dataset with Fares v1, since there
 
 <a class="button no-icon" href="https://share.mobilitydata.org/gtfs-fares-v2" target="_blank">See the full proposal</a>
 
-## In Progress Fares v2 Work
-
-<a href="https://github.com/google/transit/pull/355#issuecomment-1468326858" target="_blank">The fare media (formerly fare containers) proposal has passed</a> and been officially adopted in the spec!
-
-The Fares v2 working meetings will now focus on <a href="https://github.com/google/transit/pull/357">modelling and adopting time variable fares.</a>
-
+## Participate in the conversation
```

Comment:
Maybe we can add a simple line under this heading with something along the lines of: "You can stay up to date and join the discussions around Fares v2 by joining our Slack Chanel and the recurring working group meetings."